client/deposits/index.tsx

```diff
@@ -10,12 +10,37 @@ import React from 'react';
  */
 import Page from 'components/page';
 import { TestModeNotice } from 'components/test-mode-notice';
+import BannerNotice from 'components/banner-notice';
+import DepositSchedule from 'components/deposits-overview/deposit-schedule';
+import { useAllDepositsOverviews } from 'data';
 import DepositsList from './list';
 
+const NextDepositNotice: React.FC = () => {
+	const {
+		overviews: { account },
+	} = useAllDepositsOverviews();
+
+	const depositSettings = wcpaySettings.accountStatus?.deposits;
+	const isDepositsUnrestricted =
+		wcpaySettings.accountStatus.deposits?.restrictions ===
```

Comment:
Would this work?

```suggestion
		depositSettings?.restrictions ===
```

Reply:
Even better, remove `depositSettings`` entirely. I'm not sure why I left that in here.

Resolved in 25de881ab9695ddb149523273cc08a4eee7fa3d8